package-lock.json

```diff
@@ -3396,7 +3396,7 @@
 		},
 		"@storybook/addon-a11y": {
 			"version": "4.1.18",
-			"resolved": "https://registry.npmjs.org/@storybook/addon-a11y/-/addon-a11y-4.1.18.tgz",
+			"resolved": "https://npm.prod.pcln.com/repository/npm-all/@storybook/addon-a11y/-/addon-a11y-4.1.18.tgz",
```

Comment:
Do we want to keep the changes in this file for this PR? Might be better to do the lock update separately.

Reply:
You're right, this also made it pull from the internal repo 😬